### ISSUE TITLE
'after render' event

### ISSUE THREAD

**Issue by heyitsmeuralex** [choojs/choo#612]
I would really appreciate having an `after render` event fired once the app is re-rendered after `render` is emitted. I frequently find myself wanting to bind events and focus things after nanomorph does it's thing - the only solution I've found so far is to just setTimeout for 25ms, which feels very hacky 🚋

**Comment by bcomnes** [comment]
You can use https://github.com/choojs/nanocomponent#nanocomponentprototypeafterupdateel

To trigger events after a component updates.  That being said, a top level after render event would be handy to trigger global handlers.

**Comment by heyitsmeuralex** [comment]
Thank you! How do I access the Nanocomponent instance?

**Comment by bcomnes** [comment]
Not sure I understand the question.

Here is an example of a view using components: https://github.com/hypermodules/hyperamp/blob/master/renderer/player/pages/main.js

And the corresponding component: https://github.com/hypermodules/hyperamp/blob/master/renderer/player/elements/playlist/index.js

Your `afterupdate` method implementation will have access to `this` of the component and the raw `el` will be passed as an argument which you can use to set scroll position among other things.

**Comment by heyitsmeuralex** [comment]
I'm not using nanocomponent in my app :(  
Is there any other way I can detect once a render is complete?

**Comment by marcbachmann** [comment]
You'll need to install the `nanocomponent` module by yourself, instantiate it manually and use it in a view.

We should create a follow-up issue to document that in choo or nanocomponent. Nanocomponent could use a `load` example. Currently it's only mentioned in the api but not in an example: https://github.com/choojs/nanocomponent#nanocomponentprototypeloadel

A general question is whether we should we add examples for third party libraries to the new choo reference documentation. IMO we should at least mention nanocomponent in some documentation about more advanced use-cases.
